### PR TITLE
Fixes #365

### LINKED ISSUE #365: Improve error message when sorting without the `sort` criterion
Following this feedback https://github.com/meilisearch/MeiliSearch/discussions/1713

When doing a search with the `sort` parameter but not `sort` ranking rule is set in the `rankingRules` settings, an error is return. The current message is confusing and should be improved like this:

```
You must specify where "sort" is listed in the rankingRules settings to use the sort parameter at search time. 
```

I put it as good first issue 🙂

### PR DESCRIPTION
**Pull request by vishnugt**
Fixes #365.

### CODE REVIEW

**Comment by irevoire** [comment]
Good works!
@Kerollmops I think we can merge :+1:

File: milli/src/error.rs [hunk 1]
```diff
@@ -247,8 +247,8 @@ only composed of alphanumeric characters (a-z A-Z 0-9), hyphens (-) and undersco
                 )
             }
             Self::SortRankingRuleMissing => f.write_str(
-                "The sort ranking rule must be specified in the \
-                    ranking rules settings to use the sort parameter at search time",
+                "You must specify where \"sort\" is listed in the \
+rankingRules settings to use the sort parameter at search time",
```
Comment (Kerollmops) [inline milli/src/error.rs:251]:
```suggestion
rankingRules setting to use the sort parameter at search time",
```

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Appart fro this little thing, I am ok to merge :)
Thank you!

**Comment by vishnugt** [comment]
Updated it to "setting" @Kerollmops

**Review by Kerollmops — APPROVED** [review]
Thank you @vishnugt, merging!
Fixes #365.

bors merge

**Pull request merged**